tests/flakes.sh

```diff
@@ -27,8 +27,12 @@ flakeFollowsB=$TEST_ROOT/follows/flakeA/flakeB
 flakeFollowsC=$TEST_ROOT/follows/flakeA/flakeB/flakeC
 flakeFollowsD=$TEST_ROOT/follows/flakeA/flakeD
 flakeFollowsE=$TEST_ROOT/follows/flakeA/flakeE
+flakeHierMain=$TEST_ROOT/hier/flakeMain
+flakeHierSub=$TEST_ROOT/hier/flakeSub
```

Comment:
This "subflake" terminology is confusing. To me, a subflake is a flake in a subdirectory of a repo, not an input of another flake.